src/main/java/ua/ithillet/lesson6/WorkWithListCollection.java

```diff
@@ -0,0 +1,62 @@
+package ua.ithillet.lesson6;
+
+import java.util.*;
+
+public class WorkWithListCollection {
+
+    public static int countOccurrence(List<String> list, String searchItem) {
+        int count = 0;
+        for (String item : list) {
+            if (item.equals(searchItem)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static List<Integer> toList(Integer[] list) {
+        List<Integer> targetList = new ArrayList<>(List.of(list));
+        return targetList;
+
+    }
+
+    public static List<Integer> findUnique(List<Integer> list) {
+        List<Integer> uniqueList = new ArrayList<>();
+        try {
+            for (Integer i : list) {
+                    if (!uniqueList.contains(i)) {
+                        uniqueList.add(i);
+                    }
+            }
+        } catch (RuntimeException ex) {
+            ex.printStackTrace();
+        }
+        return uniqueList;
+    }
+
+    public static List<String> calcOccurrence(List<String> list) {
+        List<String> countUniqueWords = new ArrayList<>();
+        List<String> result = new ArrayList<>();
+        for (String str : list) {
```

Comment:
```suggestion
        for (String str : findUnique(list)) {
```
Якщо так змінити, то можна і не думати про  дублікати (тобто, countUniqueWords можна видалити)